sdk/core/core-amqp/review/core-amqp.api.md

```diff
@@ -175,10 +175,10 @@ export interface ConnectionConfig {
 }
 
 // @public
-export namespace ConnectionConfig {
-    export function create(connectionString: string, path?: string): ConnectionConfig;
-    export function validate(config: ConnectionConfig, options?: ConnectionConfigOptions): void;
-}
+export const ConnectionConfig: {
+    create(connectionString: string, path?: string | undefined): ConnectionConfig;
```

Comment:
path is an optional parameter, therefore it shouldnt be necessary to add `| undefined` to its type right?

Reply:
Interesting.. I dont see this `| undefined` in the actual function declaration, but it appears only in the api extractor output

Reply:
I think api extractor might be being pedantic here

Reply:
Rebuilding doesn't clean this, so I think that confirms that this is made by api-extractor.